sdf_timing/utils.py

```diff
@@ -1,3 +1,42 @@
+import re
+
+
+def get_scale(timescale):
```

Comment:
I think we might want a `def get_scale_ps(timescale) -> int:` (IE a version which returns int / long values).

Reply:
As soon as I created the PR I remembered I didn't like the example that gives 9.9....  I'll add the ps variant.